spec/views/posts/posts_show_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe 'User', type: :feature, js: false do
+  before :each do
+    @user = User.create(name: 'Tom', photo: 'https://randomuser.me/api/portraits/men/2.jpg',
+                        bio: 'Teacher in Mexico')
+    @first_post = Post.create(author: @user, title: 'Hello', text: 'This is my first post')
+    @comment = Comment.create(post: @first_post, author: @user, text: 'Hi Tom!')
+
+    visit "/users/#{@user.id}/posts"
+  end
+
+  describe 'Post page' do
+    it "shows the user's profile" do
+      expect(page).to have_css("img[src*='https://randomuser.me/api/portraits/men/2.jpg']")
+    end
+
+    it "shows the user's username" do
+      user_name = Post.find_by(author: @user.name)
+      expect(page).to have_content(user_name)
+    end
+
+    it 'shows the number of post' do
+      expect(page).to have_content(@user.posts_count)
+    end
```

Comment:
- [ ] @team kindly remove this test since you don't have that functionality on your post-show page.